ansible/roles/postfix/defaults/main.yml

```diff
@@ -866,15 +908,12 @@ postfix__original_mastercf:
 
       - name: 'smtpd_client_restrictions'
         value: '$mua_client_restrictions'
-        state: 'comment'
 
       - name: 'smtpd_helo_restrictions'
         value: '$mua_helo_restrictions'
-        state: 'comment'
 
       - name: 'smtpd_sender_restrictions'
         value: '$mua_sender_restrictions'
-        state: 'comment'
```

Comment:
Instead of modifying the `postfix__original_mastercf` variable which contains the values that come with the Debian `postfix` package, you should override them in the `postfix__default_mastercf` variable. This way the changes applied by DebOps should be easier to spot. You can see how it's done in the `postfix__default_maincf` variable for comparsion.

Reply:
have refactor that